script.js

```diff
@@ -89,4 +89,47 @@ document.addEventListener("DOMContentLoaded", () => {
     .catch((err) => {
       throw new Error(err.message);
     });
+
+  getData("https://jsonplaceholder.typicode.com/posts?_limit=10")
+    .then((data) =>
+      createTable(
+        "Table #2 (Posts data)",
+        ["id", "title", "body", "unique", "more than 1"],
+        data
+      )
+    )
+    .catch((err) => {
+      throw new Error(err.message);
+    });
+
+  const unique = (str, isUnique) => {
+    const arr = str.split("");
+    const objUniq = {};
+    for (let i = 0; i < arr.length; i++) {
+      if (objUniq[arr[i]]) {
+        objUniq[arr[i]] += 1;
+      } else {
+        objUniq[arr[i]] = 1;
+      }
+    }
+    const uniqueLetters = Object.values(objUniq).filter(
+      (item) => item < 2
+    ).length;
+    const moreThanOnce = arr.length - uniqueLetters;
+    if (isUnique) {
+      return uniqueLetters;
+    } else {
+      return moreThanOnce;
+    }
+  };
+
+  const createUniqTd = (name, body, clbck, item) => {
+    if (name === "unique") {
+      return clbck(body, true);
+    } else if (name === "more than 1") {
+      return clbck(body, false);
+    } else {
+      return item[name];
+    }
```

Comment:
Try to avoid `if else if else` constructions
Use switch instaed or even better use object with dynamic names